src/lib/keypom-utils.ts

```diff
@@ -314,11 +328,11 @@ const getNoneFcsAndDepositRequired = (fcData, usesPerKey) => {
 };
 
 // Estimate the amount of allowance required for a given attached gas.
-const getFtCosts = async (near, numKeys, usesPerKey, ftContract) => {
+const getFtCosts = async (near: Near, numKeys: number, usesPerKey: number, ftContract: string): Promise<string> => {
```

Comment:
we might want to use a type alias for things like this?

Balance? to match the Rust env and just readability.

WDYT?